finance_manager/src/main/resources/db/changelog/changeset/create-table-wallet.yaml

```diff
@@ -4,22 +4,22 @@ databaseChangeLog:
         author: gapserg
         changes:
           - createTable:
-              tableName: wallets
+              tableName: wallets_liq
               columns:
                 - column:
-                    name: id
+                    name: w_id
                     type: int8
                     autoIncrement: true
                     constraints:
                       primaryKey: true
                       nullable: false
                 - column:
-                    name: name
+                    name: w_name
                     type: varchar(255)
                     constraints:
                       nullable: false
                 - column:
-                    name: amount
+                    name: w_amount
                     type: numeric(20,2)
                     constraints:
                       nullable: false
```

Comment:
Pls never change already committed liqubase scripts. If script has been already committed you must create separate script.
So in your case you should create separate liqubase file which will be contain renaming wallets columns.